gadget/mountedfilesystem.go

```diff
@@ -508,20 +495,20 @@ func (f *mountedFilesystemUpdater) updateOrSkipFile(dstRoot, source, target stri
 	return writeFileOrSymlink(srcPath, dstPath, preserveInDst)
 }
 
-func (f *mountedFilesystemUpdater) updateVolumeContent(volumeRoot string, content *VolumeContent, preserveInDst []string, backupDir string) error {
+func (f *mountedFilesystemUpdater) updateVolumeContent(volumeRoot string, content *ResolvedContent, preserveInDst []string, backupDir string) error {
 	if err := checkContent(content); err != nil {
 		return err
 	}
 
-	// TODO: ResolvedSource() will already have resolved f.entrySourcePath
-	srcPath := f.entrySourcePath(content.ResolvedSource())
+	// TODO: ResolvedSource will already have resolved f.entrySourcePath
+	srcPath := f.entrySourcePath(content.ResolvedSource)
```

Comment:
can't we just get rid of this function now in this pr? if the function does nothing then I am slightly confused why we need to keep it around just to return the input exactly unchanged

Reply:
ah I see the other PR removes this function and does other clean-ups, in that case this is fine